schema.sql

```diff
@@ -0,0 +1,36 @@
+-- Table: public.books
+-- DROP TABLE IF EXISTS public.books;
+CREATE TABLE IF NOT EXISTS public.books
+(
+    id integer NOT NULL DEFAULT nextval('books_id_seq'::regclass),
+    publisher text COLLATE pg_catalog."default",
+    cover_state text COLLATE pg_catalog."default",
+    publish_date date,
+    label_id integer,
+    archived boolean,
+    CONSTRAINT books_pkey PRIMARY KEY (id),
+    CONSTRAINT label_id FOREIGN KEY (label_id)
+        REFERENCES public.labels (id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
+)
+
+TABLESPACE pg_default;
+ALTER TABLE IF EXISTS public.books
+    OWNER to postgres;
+
+
+-- Table: public.labels
+-- DROP TABLE IF EXISTS public.labels;
+CREATE TABLE IF NOT EXISTS public.labels
+(
```

Comment:
- Dear Team good job creating the `labels` and `books`, but we are missing the `music_albums`, `games`, `authors`, and `genres` tables. Please add these tables.  Also, Please make sure all the corresponding columns and associations are setup correctly.

Reply:
music_albums, games, authors, and genres schema is created in {another}-schema.sql file, otherwise, I will join all schema in this file.